R/autoplot.R

```diff
@@ -34,29 +33,29 @@
 #'   time_interval = "day"
 #' )
 #'
-#' # Create a ggplot visualization for the aedseo_tsd object
-#' autoplot(aedseo_tsd_object)
+#' # Create a ggplot visualization for the tsd object
+#' autoplot(tsd_object)
 #'
-#' # Create an aedseo object
-#' aedseo_object <- aedseo(
-#'   tsd = aedseo_tsd_object,
+#' # Create an tsd object
+#' tsd_object <- seasonal_onset(
```

Comment:
This highlights why I prefer to have functions as verbs:

Consider if instead this had been:
```
seasonal_onset <- detect_seasonal_onset(
 ...
)
```

I think this is just better since I would want to name my variable "seasonal_onset" instead of "tsd_object", but then I would overwrite the function name.

Reply:
Also above we had this issue where the function was called "tsd" and the *natural* name for the variable is also "tsd" so instead it became "tsd_object".

Consider if instead this had been:
```
# Before
tsd_object <- tsd(
  ...
)

# Alternatively
tsd <- to_tsd(
  ...
)
```